docs/feature-gates.md

```diff
@@ -209,59 +209,21 @@ there is a risk of conflicts in CIDR allocation between the two.
 
 ### AntreaIPAM
 
-`AntreaIPAM` is a feature that allows flexible control over Pod IP Addressing. This can
-be achieved by configuring custom resource `IPPool` with desired set of IP ranges.
-The pool can be annotated to Namespace, and Antrea will manage IP address assignment for
-corresponding Pods according to pool spec. In the future, support will be extended to
-Deployments and StatefulSets, as well as requesting specific IP via Pod annotation.
+`AntreaIPAM` feature allows flexible control over Pod IP addressing. This can be achieved
+by configuring custom resource `IPPool` with desired set of IP ranges. The pool can be
```

Comment:
Could you also update this doc with the new revisions in antrea-ipam.md?

Reply:
Sure. Synced the latest version to feature-gates.md